src/pages/index.tsx

```diff
@@ -1,21 +1,184 @@
-/* eslint-disable no-unused-vars */
 import * as React from "react";
-import { Link } from "gatsby";
-
 import Layout from "../components/layout";
-import Image from "../components/image";
-import SEO from "../components/seo";
+import { Table as AntTable, Typography } from "antd";
+import { ColumnsType } from "antd/es/table";
+
+interface HardwareData {
+	name: string;
+	identifier: string;
+	repository: string;
+	version: string;
+	license: string;
+	organization: string;
+	language: string;
+	tsdcId: number;
+}
+
+const tableHeaders: ColumnsType<HardwareData> = [
+	{
+		title: "Name",
+		dataIndex: "name",
+		key: "name"
+	},
+	{
+		title: "Identifier",
+		dataIndex: "identifier",
+		key: "identifier",
+		// eslint-disable-next-line react/display-name
+		render: identifier => <a>{identifier}</a>
+	},
+	{
+		title: "Repository",
+		dataIndex: "repository",
+		key: "repository"
+	},
+	{
+		title: "Version",
+		dataIndex: "version",
+		key: "version"
+	},
+	{
+		title: "License",
+		dataIndex: "license",
+		key: "license"
+	},
+	{
+		title: "Organization",
+		dataIndex: "organization",
+		key: "organization"
+	},
+	{
+		title: "Language",
+		dataIndex: "language",
+		key: "language"
+	},
+	{
+		title: "TsdcId",
+		dataIndex: "tsdcId",
+		key: "tsdcId"
+	}
+];
+
+const tableData: HardwareData[] = [
```

Comment:
All items are missing keys

Reply:
Key added to table at line 179 via rowKey,  using each entries "tsdcId" value.